README.md

```diff
@@ -274,6 +274,13 @@ Next, you'll find the following commands handy:
 - `poetry run pylint src`
 - `make docs`
 
+Please note that we currently use version poetry version 1.2.2. You can make
+sure to install this version of poetry, e.g. by running:
+
+```bash
+pipx install poetry==1.2.2
+```
+
```

Comment:
I don't think this paragraph is necessary at all. Any version of Poetry >=1.1 should work, which is well-documented in pyproject.toml.

Even if we do document something here, it should say >=1.1, not a pinned version.